### DISCUSSION TITLE
Welcome to filament-curator Discussions!

### DISCUSSION THREAD

**Discussion by awcodes** [awcodes/filament-curator#88] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by stefantheo** [comment]
Hi, i added GlobalSettings in Providers\AppServiceProvider.php, public function register, but it doesn't work.
I get error "Constant 'resourceLabel' not found in \Awcodes\Curator\Facades\Curator". 
I need to create another Provider for filament-curator?

**Reply by awcodes** [reply]
That's weird resource label isn't a constant. Can you share your code?
